src/Orchard.Web/Modules/Orchard.OpenId/Constants.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Orchard.OpenId {
+    public class Constants {
+        public const string AuthenticationErrorUrl = "~/Authentication/Error";
+        public const string LogonCallbackUrl = "~/Users/Account/LogonCallback";
+        public const string OpenIdOwinMiddlewarePriority = "10";
+        public const string LocalIssuer = "LOCAL AUTHORITY";
+        public const string FormsIssuer = "Forms";
+
+        public const string AzureActiveDirectoryObjectIdentifierKey = "http://schemas.microsoft.com/identity/claims/objectidentifier";
+
+        // Certificate Subject Key Identifier
+        public const string VeriSignClass3SecureServerCA_G2 = "A5EF0B11CEC04103A34A659048B21CE0572D7D47";
```

Comment:
I see it seems to be only for Twitter. Could we make these site settings so it can be updated without having to redeploy, or get a new Orchard deployment? It could be an advanced site setting along with the current twitter settings, hidden in a collapsible section as it's just noise. And an admin banner that would be triggered from a date that is also in these settings, based on the first certificate to expire because at some point it will stop working and people will never find out why.